pkg/bosh/manifest/kube_converter.go

```diff
@@ -889,3 +861,28 @@ func (m *Manifest) ApplyBPMInfo(kubeConfig *KubeConfig, allResolvedProperties ma
 	}
 	return nil
 }
+
+// JobSpecCopierContainer will return a v1.Container{} with the populated field
+// TODO: fix inContainerReleasePath path
```

Comment:
is that still needed?

Reply:
sorry, I was testing something, did not expect you to review asap. This will become obsolete with my next change.